CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ##### Enhancements
 
+* Add a rule that enforces alphabetical sorting of imports.  
+  [Scott Berrevoets](https://github.com/sberrevoets)
+  [#991](https://github.com/realm/SwiftLint/pull/991)
```

Comment:
reference the issue number this solves (#900): https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes